src/app/modules/visualizacion-informacion/components/bi-queries/bi-queries.component.ts

```diff
@@ -24,6 +24,10 @@ export class BiQueriesComponent implements OnInit {
     public mostrarSalida = false;
     public tipoPrestaciones;
     public totalOrganizaciones = false;
+    public opcionesSexo = [
```

Comment:
Veo que vienen haciendo un montón de PR cada vez que necesitan una nueva "tabla" de datos. Poruqe no agregan esos valores en la DB?

<plex-select *ngSwitchCase="'select-static'"
```suggestion
    public opcionesSexo = [
                                     [label]="argumento.label?argumento.label:argumento.key" grow=" 2"
                                     name="{{ argumento.key }}" [(ngModel)]="argumentos[argumento.key]"
                                     [required]="argumento.required" labelField="nombre" 
                                     [data]="argumento.items">
```